test/support/api/organizations.js

```diff
@@ -0,0 +1,35 @@
+import _ from 'underscore';
+import { getResource, getIncluded, getRelationship } from 'helpers/json-api';
+
+Cypress.Commands.add('routeOrganization', (mutator = _.identity) => {
+  cy
+    .fixture('collections/organizations').as('fxOrganizations')
+    // .fixture('collections/clinicians').as('fxClinicians')
```

Comment:
I do suspect at some point, we'll need to be loading the clinicians for the groups as it'll take all of that info to make the assign component, but I'm ok with this for now

Reply:
oh yeah...I didn't see this as a relationship on the org endpoint, so I just left this here.